databusclient/client.py

```diff
@@ -25,6 +28,94 @@ class DeployLogLevel(Enum):
     info = 1
     debug = 2
 
+# Define constants for different identifiers
+GROUP_IDENTIFIER = "group"
+ARTIFACT_IDENTIFIER = "artifact"
+VERSION_IDENTIFIER = "version"
+FILE_IDENTIFIER = "file"
+USER_IDENTIFIER = "user"
+
+def __download_files(urls: List[str], local_dir: str):
+    for url in urls:
+        __download_file__(url=url, filename=os.path.join(local_dir, wsha256(url)))
+
+def download(
```

Comment:
the function overrides the download below in the code, it is not editing of the old function function, but writing a new one in the same file with the old one on line 461 in old version